src/interfaces/express.ts

```diff
@@ -42,9 +46,9 @@ export function makeExpressHandler(client, config) {
    * @param {Function} next - an Express continuation callback
    * @returns {ErrorMessage} - Returns the ErrorMessage instance
    */
-  function expressErrorHandler(err, req, res, next) {
+  function expressErrorHandler(err: {}, req: express.Request, res: express.Response, next: Function) {
     var ctxService = '';
-    var ctxVersion = '';
+    var ctxVersion: string|undefined = '';
```

Comment:
`var` again.  Also, can we use the `let ctxVersion?: string` syntax?

Reply:
AFAIK that syntax doesn't work here.  It only works for function parameter annotations.